tests/test_metadata_extractors.py

```diff
@@ -311,7 +311,7 @@ def setup_method(self, method):
                            ['BuildRequires', 'python-setuptools'],
                            ['BuildRequires', 'python-sphinx']]),
         (1, 'py_modules', ['pytest']),
-        (1, 'packages', ['_pytest']),
+        # (1, 'packages', ['_pytest', 'pytest']), # py34 will get setuptools 43, and packages will not have 'pytest'
```

Comment:
Can you conditionally enable/disable this test depending on the python version?

Reply:
I've pushed a change to do that.